bitcoin/src/lib.rs

```diff
@@ -84,7 +84,9 @@ pub extern crate io;
 #[cfg(feature = "ordered")]
 pub extern crate ordered;
 
-/// Rust wrapper library for Pieter Wuille's libsecp256k1.  Implements ECDSA and BIP 340 signatures
+/// Re-export the `rust-secp256k1` crate.
+///
+/// Rust wrapper library for Pieter Wuille's libsecp256k1. Implements ECDSA and BIP-340 signatures
```

Comment:
```suggestion
/// Rust wrapper library for Pieter Wuille's libsecp256k1. Implements ECDSA and BIP-340 signatures.
```

Forgot to end the sentence with `.`.